src/acceptance-test/java/de/zalando/aruha/nakadi/repository/db/EventTypeDbRepositoryTest.java

```diff
@@ -108,7 +109,7 @@ public void whenUpdateExistingEventTypeItUpdates() throws NakadiException, Dupli
 
         repository.saveEventType(eventType);
 
-        eventType.setCategory("new-category");
+        eventType.setCategory(EventCategory.UNDEFINED);
```

Comment:
I mean, if we are testing updates, than we should change something in order for our assertions have some value. Since we are not changing the original value, then this test is testing nothing.